src/plugins/intel_cpu/src/ngraph_transformations/rnn_sequences_optimization.cpp

```diff
@@ -57,7 +57,7 @@ namespace {
             ngraph::Output<ngraph::Node> in_0 = sequenceOp->get_input_node_shared_ptr(0)->input_value(0);
 
             auto shapeBeforeTranspose = ngraph::op::util::make_try_fold<ngraph::opset1::ShapeOf>(in_0);
-            auto newInShape = ngraph::op::util::make_try_fold<ngraph::opset7::Gather>(shapeBeforeTranspose,
+            auto newInShape = ngraph::op::util::make_try_fold<ngraph::opset8::Gather>(shapeBeforeTranspose,
```

Comment:
Please use ov namespace instead of ngraph due to second is deprecated.

Reply:
I changed the namespace where possible, thank you. But the utils are still in ngraph namespace